test/integration/auth.spec.ts

```diff
@@ -1333,6 +1456,9 @@ describe('admin.auth', () => {
         ],
       },
       emailPrivacyConfig: {},
+      mobileLinksConfig: {
```

Comment:
can we also add a set of tests if it was not already implemented
When domain is "FIREBASE_DYNAMIC_LINK_DOMAIN", it should generated default fdl domain link.
When domain mobileLinksConfig = {}, it should generated default fdl domain link.

cc @lahirumaramba  do we have a default FDL domain to use?

Reply:
Added testcases for the 2 scenarios, used one of the fdl generated param to assert it is FDL link.